### PR TITLE
fix(tip-manager): Set padding for tips and tip-groups consistently

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #6464

## Summary

- Removes use of `:host-context` CSS selector. https://developer.mozilla.org/en-US/docs/Web/CSS/:host-context
  -  It's not supported in FF or Safari. @jcfranco can be ban its usage?
- Sets padding inside the tip manager container instead of on slotted elements for consistency.
- Sets tip max-width on tips within the `calcite-tip-manager` and within `calcite-tip-group`'s
- Adds screenshot tests
- Fixes link to tooltip manager html file in index of demos

### CODE REVIEW

File: src/components/tip-group/tip-group.scss [hunk 1]
```diff
@@ -8,12 +8,5 @@
 
 ::slotted(calcite-tip) {
   @apply m-0 border-none;
-}
-
-:host-context(calcite-tip-manager) {
-  @apply my-3;
-}
-
-:host-context(calcite-tip-manager) ::slotted(calcite-tip) {
```
Comment (driskull) [inline src/components/tip-group/tip-group.scss:17]: Can we setup a lint error/warning for using host-context so its not checked into any files?

File: src/components/tip-manager/tip-manager.scss [hunk 2]
```diff
@@ -61,7 +61,8 @@
     flex
     items-start
     justify-center
-    overflow-auto;
+    overflow-auto
+    p-4;
```
Comment (driskull) [inline src/components/tip-manager/tip-manager.scss:65]: Used `p-4` to line up with header

File: src/components/tip-group/tip-group.scss (hunk 1 above)
Comment (jcfranco) [reply src/components/tip-group/tip-group.scss:17]: Good idea. I'll see what we can set up for this.

**Review by jcfranco — APPROVED** [review]
💥

**Comment by driskull** [comment]
@ashetland @macandcheese can you review the screenshot changes for this PR?

**Pull request merged**